src/unity/python/turicreate/toolkits/object_detector/_sframe_loader.py

```diff
@@ -149,6 +149,10 @@ def _next(self):
             oshape = orig_image.shape
             if self.load_labels:
                 label = row[self.annotations_column]
+                if label == None:
+                    label = []
+                if type(label) == dict:
```

Comment:
How about an `elif` here? Whenever I see if statements like this I assume both can occur and waste a few cycles realizing they can't.

Reply:
Done